runtime/queries/haxe/highlights.scm

```diff
@@ -0,0 +1,71 @@
+(identifier) @type
```

Comment:
This needs to be moved to the bottom of the file: when it's at the top it takes precedence over all other uses of `(identifier)` so everything looks like a type.

(This is because neovim queries are in reverse order and these queries are in neovim format)